src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -221,30 +235,36 @@ $pf-v6--include-toolbar-breakpoints: true !default;
 
 // - Toolbar group
 .#{$toolbar}__group {
-  row-gap: var(--#{$toolbar}__group--RowGap, var(--#{$toolbar}--RowGap));
-  column-gap: var(--#{$toolbar}__group--ColumnGap, var(--#{$toolbar}--ColumnGap));
-  align-items: baseline;
-
-  // - Toolbar icon button group - Toolbar filter group
-  &.pf-m-icon-button-group {
-    column-gap: var(--#{$toolbar}__group--m-icon-button-group--ColumnGap, 0);
-  }
+  row-gap: var(--#{$toolbar}__group--RowGap);
+  column-gap: var(--#{$toolbar}__group--ColumnGap);
 
+  // - Toolbar filter group
   &.pf-m-filter-group {
     column-gap: var(--#{$toolbar}__group--m-filter-group--ColumnGap);
   }
 
-  // - Toolbar filter group - Toolbar label container
-  &.pf-m-filter-group,
-  &.pf-m-label-group-container {
-    flex-wrap: nowrap;
-  }
-
   // - Toolbar label group
   &.pf-m-label-group {
     flex: 1;
+    column-gap: var(--#{$toolbar}__group--m-label-group--ColumnGap);
   }
 
+  // - Toolbar action group
+  &.pf-m-action-group {
+    column-gap: var(--#{$toolbar}__group--m-action-group--ColumnGap);
+  }
+
+  // - Toolbar action group plain
+  &.pf-m-action-group-plain {
+    column-gap: var(--#{$toolbar}__group--m-action-group-plain--ColumnGap);
+  }
+
+  // - Toolbar action group inline
+  &.pf-m-action-group-inline {
+    column-gap: var(--#{$toolbar}__group--m-action-group-inline--ColumnGap);
```

Comment:
not sure if this would be considered a action to action or a gap group horizontal? @lboehling need some assistance here! 
<img width="645" alt="Screenshot 2024-07-03 at 4 39 46 PM" src="https://github.com/patternfly/patternfly/assets/150823809/c98b33fe-3ee4-4fc8-a58d-3b1e1f8d4555">
<img width="599" alt="Screenshot 2024-07-03 at 4 40 04 PM" src="https://github.com/patternfly/patternfly/assets/150823809/44e4c5a4-b70f-4a1c-93eb-41b2e76a645d">